armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -64,26 +70,25 @@ def expandColdDimsToHot(assems, isDetailedAxialExpansion, referenceAssembly=None
     assems: list[:py:class:`Assembly <armi.reactor.assemblies.Assembly>`]
         list of assemblies to be thermally expanded
     isDetailedAxialExpansion: bool
-        If true assemblies will be forced to conform to the reference mesh after expansion
+        If False, assemblies will be forced to conform to the reference mesh after expansion
+    assemsToSkip: List[str]
+        list of strings to be converted to flags to indicate which assemblies to skip axial expansion
     referenceAssembly: :py:class:`Assembly <armi.reactor.assemblies.Assembly>`, optional
-        Assembly whose mesh other meshes wil conform to if isDetailedAxialExpansion is true.
+        Assembly whose mesh other meshes will conform to if isDetailedAxialExpansion is False.
         If not provided, will assume the finest mesh assembly which is typically fuel.
     """
+    aToSkip = list(Flags.fromStringIgnoreErrors(t) for t in assemsToSkip)
     assems = list(assems)
     if not referenceAssembly:
         referenceAssembly = getDefaultReferenceAssem(assems)
     axialExpChanger = AxialExpansionChanger(isDetailedAxialExpansion)
     for a in assems:
-        if not a.hasFlags(Flags.CONTROL):
-            axialExpChanger.setAssembly(a)
-            # this doesn't get applied to control assems, so CR will be interpreted
-            # as hot. This should be conservative because the control rods will
-            # be modeled as slightly shorter with the correct hot density. Density
-            # is more important than height, so we are forcing density to be correct
-            # since we can't do axial expansion (yet)
-            axialExpChanger.applyColdHeightMassIncrease()
-            axialExpChanger.expansionData.computeThermalExpansionFactors()
-            axialExpChanger.axiallyExpandAssembly()
+        if a.hasFlags(Flags.CONTROL) or any(a.hasFlags(aFlags) for aFlags in aToSkip):
+            continue
```

Comment:
?

https://github.com/terrapower/armi/pull/1226

Reply:
Once #1226 is merged in, the check on `a.hasFlags(Flags.CONTROL)` will be removed. I left it in there to keep the PRs independent of one another.

Reply:
Just take that part out now and put a comment on this PR saying not to merge until #1226 is merged?

Reply:
Like, don't delete all those lines either. Github will handle the diffs properly. Then this PR will actually be independent of that one.

Reply:
I'll just pop off the check on `a.hasFlags(Flags.CONTROL)`.

Reply:
I'm an idiot, ignore this comment thread!